ios_app_clip/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -217,8 +218,8 @@
 				97C146EB1CF9000F007C117D /* Frameworks */,
 				97C146EC1CF9000F007C117D /* Resources */,
 				9705A1C41CF9048500538489 /* Embed Frameworks */,
-				3B06AD1E1E4923F5004D2608 /* Thin Binary */,
 				0A48A26E2508B57F0036A529 /* Embed App Clips */,
+				3B06AD1E1E4923F5004D2608 /* Thin Binary */,
```

Comment:
Rearrange these to avoid a cycle error caused by `Runner.app/Info.plist`:
```
Cycle inside Runner; building could produce unreliable results.
Cycle details:
→ Target 'Runner' has copy command from 'Runner.app/AppClips/AppClip.app'
○ That command depends on command in Target 'Runner': script phase “Thin Binary”
○ Target 'Runner' has process command with output 'Runner.app/Info.plist'
○ Target 'Runner' has copy command from 'Runner.app/AppClips/AppClip.app'
```